docs/en/Clients-Go.md

```diff
@@ -39,18 +39,51 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"log"
+	"strings"
 
-	alluxio "github.com/Alluxio/alluxio-go"
-	"github.com/Alluxio/alluxio-go/option"
+	alluxio "github.com/alluxio/alluxio-go"
+	"github.com/alluxio/alluxio-go/option"
 )
 
 func main() {
-	fs := alluxio.NewClient(<proxy-host>, <proxy-port>, <timeout>)
-	ok, err := fs.Exists("/test_path", &option.Exists{})
+	fs := alluxio.NewClient(<proxy host>, <proxy port (39999 by default)>, <timeout>)
+	path := "/test_path"
+	exists, err := fs.Exists(path, &option.Exists{})
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Println(ok)
+	if exists {
+		if err := fs.Delete(path, &option.Delete{}); err != nil {
+			log.Fatal(err)
+		}
+	}
+	// Write data
+	id, err := fs.CreateFile(path, &option.CreateFile{})
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
`defer fs.Close(id)`?

Reply:
I wasn't sure whether to do this since we also need to close it on line 70 before we can read. Do you know the idiomatic way to handle this situation in Go?

Reply:
make sense, I think we can have two functions for write and read separately, in the function for write, the `defer` will always be executed, if the write errors, then the write function returns an error to the main function.

Reply:
done